### PR TITLE
Fixed empty types section shown in quick info

### PR DESCRIPTION
**Pull request by DoctorKrolic**
Fixes: https://github.com/dotnet/roslyn/issues/61320

### CODE REVIEW

File: src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AnonymousTypes.cs [hunk 1]
```diff
@@ -33,12 +33,10 @@ where part.Symbol.IsAnonymousType() || part.Symbol.IsTupleType()
...
 
                 if (info.TypesParts.Count > 0)
-                {
                     AddToGroup(SymbolDescriptionGroups.StructuralTypes, info.TypesParts);
 
-                    foreach (var (group, parts) in _groupMap.ToArray())
-                        _groupMap[group] = info.ReplaceStructuralTypes(parts, _semanticModel, _position);
-                }
+                foreach (var (group, parts) in _groupMap.ToArray())
+                    _groupMap[group] = info.ReplaceStructuralTypes(parts, _semanticModel, _position);
```
Comment (CyrusNajmabadi) [inline src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AnonymousTypes.cs:39]: was this change needed as well?  thanks!

Comment (DoctorKrolic) [reply src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AnonymousTypes.cs:39]: Yes. If you don't do that, you will get this (first example in the issue):
```cs
Foo(default);

void Foo(<tuple> foo)
{
}
```

**Comment by CyrusNajmabadi** [comment]
Thanks! :)

**Comment by DoctorKrolic** [comment]
@CyrusNajmabadi You've forgotten to approve again)

**Review by CyrusNajmabadi — APPROVED** [review]
I need to write a bot taht takes my comments and converts them into an approval.
